node/src/components/event_stream_server/deploy_getter.rs

```diff
@@ -67,7 +66,7 @@ impl DeployGetter {
         #[cfg(test)]
         return self.get_test_deploy(deploy_hash);
 
-        let deploy_hashes = smallvec![deploy_hash];
+        let deploy_hashes = vec![deploy_hash];
```

Comment:
What was wrong with `smallvec!`? It's an optimization where it won't make as many allocations as `Vec` for collections of less than 2 elements.

Reply:
The effect builder was just calling `to_vec()` on it anyway.